timeseries/setup.py

```diff
@@ -27,7 +27,7 @@
     "numpy",
     "scipy",
     "pandas",
-    "psutil>=5.7.3,<5.9",
+    "psutil>=5.7.3,<6",
```

Comment:
`psutil>=5.7.3,<5.10` will be safer. We generally avoid capping at major versions as it is less stable than minor version capping.

Reply:
Actually, it seems that we don't use `psutil` anywhere in `timeseries`, so it's fine to remove this dependency completely from `timeseries/setup.py`.